.github/workflows/TagBot.yml

```diff
@@ -4,6 +4,11 @@ on:
     types:
       - created
   workflow_dispatch:
+    inputs:
+      lookback:
+        default: 3
+permissions:
+  contents: write
```

Comment:
FWIW, this actually only affects the default `GITHUB_TOKEN` secret. When we add a valid `DOCUMENTER_KEY` to the repo settings, we're setting up an SSH key permission, thus this entry won't work here.

Nevertheless, it's not harmful to add it here.